test-network-function/identifiers/identifiers.go

```diff
@@ -323,7 +323,10 @@ the same hacks.'`),
 		Description: formDescription(TestOperatorInstallStatusIdentifier,
 			`Ensures that CNF Operators abide by best practices.  The following is tested:
 1. The Operator CSV reports "Installed" status.
-2. TODO: Describe operator scc check.`),
+2. Check that the operator is not configured with privileged rights.
+If clusterPermissions exists in the CSV,
+and the clusterPermissions[].rules[].resourceNames is not empty then we fail the test.
+Otherwise, the test passes. In case the test fails, we need to analyze the need for the privilege mode.`),
```

Comment:
In case the test fails, we need to analyze the need for the privilege mode.
if the test fail and you have recommendation, then it should go in remediation part.
I did not like writing exactly what the test is doing using If else, this should be a general description of the test and what it aims to check, not how it does the check exactly.

Reply:
I made the same point in the other comment. Also, avoid using "we" as it feels informal.

Reply:
I made the same point in other comment. Also, avoid using "we". It feels informal. Describe it in an objective way if we need to, e.g.: Test passes if [clusterPermissions is not present in the CSV manifest or is present with no resourceNames under its rules].
Also agree on Salah's comment on the remediation part. In this case though, it's fairly obvious what should be checked. I am actually ok with the existing generic remediation text. Just remove the last sentence.

Reply:
Ok, So I will change the description -  

Check that the operator is not configured with privileged rights.
If clusterPermissions exists in the CSV,
and the clusterPermissions[].rules[].resourceNames is not empty then fail the test.
Otherwise, the test passes.

it's OK?

Reply:
As discussed in the team channel, I am updating the text so this can be part of 3.1.